src/plugins/kibana/server/lib/create_mappings_from_pattern_fields.js

```diff
@@ -15,7 +15,7 @@ module.exports = function createMappingsFromPatternFields(fields) {
       mapping = {
         type: 'text',
         fields: {
-          raw: {type: 'keyword', ignore_above: 256}
+          keyword: {type: 'keyword', ignore_above: 256}
```

Comment:
It's the default for a dynamically mapped string, but here we're creating an explicit mapping for a known string field.